pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -34,19 +36,20 @@ const (
 	// HighPerformanceRuntime contains the name of the high-performance runtime
 	HighPerformanceRuntime = "high-performance"
 
-	hugepagesAllocation = "hugepages-allocation"
-	bashScriptsDir      = "/usr/local/bin"
-	crioConfd           = "/etc/crio/crio.conf.d"
-	crioRuntimesConfig  = "99-runtimes.conf"
-	ociHooks            = "low-latency-hooks"
+	bashScriptsDir     = "/usr/local/bin"
+	crioConfd          = "/etc/crio/crio.conf.d"
+	crioRuntimesConfig = "99-runtimes.conf"
 	// OCIHooksConfigDir is the default directory for the OCI hooks
 	OCIHooksConfigDir = "/etc/containers/oci/hooks.d"
 	// OCIHooksConfig file contains the low latency hooks configuration
-	OCIHooksConfig     = "99-low-latency-hooks"
+	OCIHooksConfig     = "99-low-latency-hooks.json"
```

Comment:
is this the only json file that now needs a specification for the extension ?   how will this play with cnf-depoly ?

Reply:
hm, what do you mean, it is the file that we are injecting under the node to make OCI hooks work.

Reply:
ack. I see its already done for the rest here for the injection (they already had the files extension before)